includes/discovery/vrf.inc.php

```diff
@@ -259,6 +259,28 @@
                 continue;
             }
         }
+    } elseif ($device['os'] == 'cumulus') {
+        // Cumulus Linux
+        $vrf_table = \SnmpQuery::hideMib()->walk(['CUMULUS-BGPVRF-MIB::bgpVrfId', 'CUMULUS-BGPVRF-MIB::bgpVrfName'])->table(1);
+        foreach ($vrf_table as $vrf_oid => $vrf_data) {
+            $vrf_name = $vrf_data['bgpVrfName'];
+
+            $vrfs = [
+                'vrf_oid' => $vrf_oid,
+                'vrf_name' => $vrf_name,
+                'device_id' => $device['device_id'],
+            ];
+
+            if (DeviceCache::getPrimary()->vrfs()->select('vrf_id')->where('vrf_oid', $vrf_oid)->count()) {
+                DeviceCache::getPrimary()->vrfs()->where('vrf_oid', $vrf_oid)->update($vrfs);
+            } else {
+                DeviceCache::getPrimary()->vrfs()->create($vrfs)->vrf_id;
```

Comment:
@laf this is broken, you're not adding `mplsVpnVrfDescription` and it doesn't have a default value

```
6) LibreNMS\Tests\OSModulesTest::testOS with data set "cumulus_bgp" ('cumulus', 'bgp')
Illuminate\Database\QueryException: SQLSTATE[HY000]: General error: 1364 Field 'mplsVpnVrfDescription' doesn't have a default value (Connection: testing, SQL: insert into `vrfs` (`vrf_oid`, `vrf_name`, `device_id`) values (254, default, 259))

includes/discovery/vrf.inc.php:278
includes/discovery/functions.inc.php:171
LibreNMS/Util/ModuleTestHelper.php:652
tests/OSModulesTest.php:106
```